projects/storefrontlib/src/layout/a11y/skip-link/service/skip-link.service.ts

```diff
@@ -0,0 +1,67 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+import { SkipLinkConfig, SkipLinkScrollPosition } from '../config/index';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class SkipLinkService {
+  skippers = new BehaviorSubject([]);
+
+  constructor(protected config: SkipLinkConfig) {}
+
+  add(key: string, target: HTMLElement) {
```

Comment:
we agreed on explicit return types, should be `:void` in this case. Please go through the code once more with this in mind.

Reply:
"we agreed" – means: we as a team had a discussion about this a long time ago... ;)